deploy/docker-compose.yaml

```
@@ -131,3 +142,20 @@ services:
         condition: service_healthy
       mongodb:
         condition: service_healthy
+  trips:
+    container_name: trips-ms
+    image: ${TRIPS_IMAGE}
+    ports:
+      - "8000:8080"
```

Na zewnątrz composa trip ms będzie dostępny dla localhosta pod portem `8000` ale wewnątrz samego klastra dla innych serwisów pod portem `8080`. Trzeba o tym pamiętać albo zmienić żeby `HOST_PORT` oraz `CONTAINER_PORT` były te same. Proponuje zmienić w springu startup port na :

```suggestion
      - "8000:8000"
```